src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -28,6 +28,7 @@ import GetAccessibilityRequestAccessibilityTeamOnlyQuery from 'queries/GetAccess
 import GetAccessibilityRequestQuery from 'queries/GetAccessibilityRequestQuery';
 import {
   CreateAccessibilityRequestNote,
+  CreateAccessibilityRequestNote_createAccessibilityRequestNote_userErrors, // eslint-disable-line camelcase
```

Comment:
You can [alias](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#rename_multiple_exports_during_import) imports